docs/operations.md

```diff
@@ -5,6 +5,8 @@ EventStoreDB requires regular maintenance with two operational concerns:
 - [Scavenging](#scavenging-events) for freeing up space after deleting events
 - [Backup and restore](#backup-and-restore) for disaster recovery
 
+In rare cases, you may also need [Redaction](#redaction) as a last resort to blank out sensitive events if [Scavenging](#scavenging-events) cannot be used.
```

Comment:
```suggestion
```

Perhaps we can leave this bit out. They shouldn't ever 'need' redaction, and they might also wonder why scavenging possibly can't be used. Since its a last resort rather than normal operational maintenance its probably enough to just have the section below